LibraBFT/ImplShared/Consensus/Types.agda

```diff
@@ -185,18 +185,25 @@ module LibraBFT.ImplShared.Consensus.Types where
   record BootstrapInfo : Set where
     constructor mkBootstrapInfo
     field
-      -- TODO-1 : Nodes, PKs for initial epoch
-      -- TODO-1 : Faults to tolerate (or quorum size?)
-      bootstrapQC : QuorumCert -- use the same bootstrap QC for highestQC and highestCommitCert
-  open BootstrapInfo
+      _bsiNumFaults : ℕ
+      _bsiLIWS      : LedgerInfoWithSignatures
+      _bsiVSS       : List ValidatorSigner -- TODO-1 : remove this after ValidatorSigner
```

Comment:
This comment is unclear.  What is "this"?  Remove it "after" what?

Reply:
I removed the comment.
What it is talking about no longer exists.